pkg/controller/stacks/install/installjob.go

```diff
@@ -122,6 +122,12 @@ func prepareInstallJob(p prepareInstallJobParams) *batchv1.Job {
 									MountPath: "/ext-pkg",
 								},
 							},
+							Env: []v1.EnvVar{
+								{
+									Name:  "STACK_IMAGE",
```

Comment:
could this not use the same const `StackImageEnv`?

Reply:
Hmm good point

Reply:
This is changed; resolving